wlog/auditlog/audit2log/params.go

```diff
@@ -112,3 +112,27 @@ func ResultParams(resultParams map[string]interface{}) Param {
 		entry.AnyMapValue(ResultParamsKey, resultParams)
 	})
 }
+
+func SafeParam(key string, value interface{}) Param {
```

Comment:
The audit.2 format doesn't contain safe and unsafe param maps (rather, dynamic content is in the request/result fields) so we shouldn't be adding these extra params.